builtin/types/user_deployer_whitelist/user_deployer_whitelist.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+package user_deployer_whitelist;
+import "github.com/loomnetwork/go-loom/types/types.proto";
+import "github.com/loomnetwork/go-loom/builtin/types/deployer_whitelist/deployer_whitelist.proto";
+import "github.com/loomnetwork/go-loom/vm/vm.proto";
+
+enum TierID {
+    DEFAULT = 0;
+}
+
+message Tier {
+    TierID id = 1;
+    uint64 fee = 2;
```

Comment:
Any particular reason this isn't a `BigUInt`?

Reply:
I discussed with Parth, was getting json unmarshalling error in case of BigUInt